picasso/src/main/java/com/squareup/picasso/Picasso.java

```diff
@@ -509,11 +509,11 @@ Bitmap quickMemoryCacheCheck(String key) {
   }
 
   void complete(BitmapHunter hunter) {
-    Action single = hunter.getAction();
+    Action action = hunter.getAction();
```

Comment:
This should remain as "single". Hunters have a "single" action and also "joined" actions.